### PR TITLE
YCDOCIO-1340: MES TF Snapshots

### PR DESCRIPTION
**Pull request by voloschuk-fedor**
@redalka-cloud , привет! Прошу провести ревью в рамках задачи [YCDOCIO-1340](https://st.yandex-team.ru/YCDOCIO-1340).

### CODE REVIEW

File: tutorials/terraform/mes-migration.tf [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Infrastructure for Yandex Cloud Managed Service for Elasticsearch cluster
+#
+# RU: https://cloud.yandex.ru/docs/managed-elasticsearch/tutorials/migration-via-snapshots
+# EN: https://cloud.yandex.com/en/docs/managed-elasticsearch/tutorials/migration-via-snapshots
+
+# Specify the pre-installation parameters
+locals {
+  folder_id          = "" # Your Folder ID.
```
Comment (redalka-cloud) [inline tutorials/terraform/mes-migration.tf:8]: Your folder ID.

File: tutorials/terraform/mes-migration.tf [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  mes_admin_password = "" # Administrator password for Managed Service for Elasticsearch cluster.
```
Comment (redalka-cloud) [inline tutorials/terraform/mes-migration.tf:9]: Administrator password for the Managed Service for Elasticsearch cluster.

File: tutorials/terraform/mes-migration.tf [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  mes_edition        = "" # Managed Service for Elasticsearch destination cluster edition, Basic or Platinum. See https://cloud.yandex.ru/docs/managed-elasticsearch/concepts/es-editions.
```
Comment (redalka-cloud) [inline tutorials/terraform/mes-migration.tf:10]: The Managed Service for Elasticsearch destination cluster edition, Basic or Platinum. See https://cloud.yandex.ru/docs/managed-elasticsearch/concepts/es-editions.

File: tutorials/terraform/mes-migration.tf [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+  mes_version        = "" # Managed Service for Elasticsearch destination cluster version, should be equal or newer than source cluster version.
```
Comment (redalka-cloud) [inline tutorials/terraform/mes-migration.tf:11]: The Managed Service for Elasticsearch destination cluster version, should be equal or newer than the source cluster version.

File: tutorials/terraform/mes-migration.tf [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+  bucket_name        = "" # Object Storage bucket name. Should be unique in Cloud.
```
Comment (redalka-cloud) [inline tutorials/terraform/mes-migration.tf:12]: The Object Storage bucket name. Should be unique in the Cloud.

File: tutorials/terraform/mes-migration.tf (hunk 1 above)
Comment (AnatolyKraev) [reply tutorials/terraform/mes-migration.tf]: Ок

File: tutorials/terraform/mes-migration.tf (hunk 2 above)
Comment (AnatolyKraev) [reply tutorials/terraform/mes-migration.tf]: Ок

File: tutorials/terraform/mes-migration.tf (hunk 3 above)
Comment (AnatolyKraev) [reply tutorials/terraform/mes-migration.tf]: Ок

File: tutorials/terraform/mes-migration.tf (hunk 4 above)
Comment (AnatolyKraev) [reply tutorials/terraform/mes-migration.tf]: Ок

File: tutorials/terraform/mes-migration.tf (hunk 5 above)
Comment (AnatolyKraev) [reply tutorials/terraform/mes-migration.tf]: Ок

**Pull request merged**